src/services/client.service.ts

```diff
@@ -14,7 +14,7 @@ export class ClientService extends Client {
         Intents.FLAGS.GUILD_VOICE_STATES,
       ],
     });
-    this.login(process.env.DISCORD_TOKEN);
+    this.login(process.env.DISCORD_TOKEN).catch(e => console.log(e));
```

Comment:
I was just using console.log to debug
But since this wont make it into production I dont think it matters? @c650

Reply:
Fine w me